driver/normalizer/strconv.go

```diff
@@ -11,6 +11,7 @@ import (
 
 // unquoteSingle is the same as strconv.Unquote, but uses ' as a quote.
 func unquoteSingle(s string) (string, error) {
+	s = s[1 : len(s)-1]
```

Comment:
length checks? checks that those trimmed chars are correct quotes?

Reply:
so far this is just a refactoring - this code was moved from the client of this method in Op impl.
So nothing changed yet.

But I know what you mean :) will try to follow exactly the same logic as is in `strconv.Unquote`

Reply:
addressed in 7d784b9be3628f8b3cc68a6e1cd34d6c495c31be